app/components/rdf-input-fields/mandataris-rangorde.js

```diff
@@ -42,7 +44,16 @@ export default class RdfMandatarisRangorde extends InputFieldComponent {
 
       this.checkIfShouldRender();
     };
-    this.storeOptions.store.registerObserver(onFormUpdate);
+    this.storeOptions.store.registerObserver((formChange) => {
+      const mustTrigger = isPredicateInObserverChange(
+        formChange,
+        MANDATARIS_PREDICATE.mandaat
+      );
+
+      if (mustTrigger) {
+        onFormUpdate();
+      }
+    });
     onFormUpdate();
```

Comment:
Should we make it a method outside the `registerObserver()` method?

Reply:
It could be abstracted a bit, but not really sure that is worth it.